src/components/DataVisualCard/useStyles.js

```diff
@@ -0,0 +1,39 @@
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles(({ typography, breakpoints }) => ({
+  root: {
+    maxWidth: typography.pxToRem(350),
+    boxShadow: "none",
+    backgroundColor: "unset",
+    [breakpoints.up("md")]: {
+      maxWidth: typography.pxToRem(296),
+    },
+    [breakpoints.up("lg")]: {
+      maxWidth: typography.pxToRem(376),
+    },
+  },
```

Comment:
`max-width` is limiting the card responsiveness. This is on mobile (webpage)
![Screenshot from 2021-07-29 09-11-07](https://user-images.githubusercontent.com/7962097/127440840-e3cc334a-1c6e-49b7-ab9f-e40d9651ff34.png)